browser/src/control/Toolbar.js

```diff
@@ -551,11 +551,14 @@ L.Map.include({
 		}
 		var searchInput = document.getElementById('online-help-search-input');
 		searchInput.focus(); // auto focus on user input field
+		var helpContentParent = document.getElementsByClassName('ui-dialog-content')[0];
+		searchInput.focus();
```

Comment:
This was badly/blindly rebased.

Reply:
@Darshan-upadhyay1110 ^ it seems you push badly formed commits. Don't push nothing else. I will fix it locally and force push

Reply:
or maybe it was my fault, I just see that we have 2 `searchInput.focus();` which evidently makes no sense